apps/designer/app/designer/features/style-panel/shared/property-name.tsx

```diff
@@ -28,7 +28,6 @@ export const PropertyName = ({ property, label, css }: PropertyProps) => {
                   color: "$hiContrast",
                 }),
           }}
-          htmlFor={property}
```

Comment:
that was for clicking on property to result in focusing the field, why removing?

Reply:
it didn't work? then we need to fix it?